templates/front-end/support.php

```diff
@@ -11,7 +11,31 @@
 ?>
 
 	<div id="primary" class="m-layout-membership o-support">
+
 		<main id="main" class="site-main" role="main">
+
+			<div class="m-support-progress" aria-label="progress">
```

Comment:
On the Flask app, the progress bar is outside the `<main>` content, which I _think_ aligns with the intended use of the `<main>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#Usage_notes

...thoughts?

Reply:
You're totally right. Probably is what my previous self was thinking. I'll restore that.